Makefile

```diff
@@ -0,0 +1,29 @@
+# Required for merging:
+
+pip:
+	pip install -r requirements.txt
+	pip install -e .
+
+build-and-test: build black mypy test
+
+build:
+	python -m scripts.generate_init --check
+
+black:
+	black --check .
+
+mypy:
+	mypy pyteal
+
+test:
+	pytest
+
+
+# Extras:
+
+coverage:
+	pytest --cov-report html --cov=pyteal
+
+blackbox:
```

Comment:
@tzaffi `make blackbox` is failing due to a change in py-algorand-sdk.  I opened https://github.com/algorand/pyteal/pull/258 to fix the failures.  Can you review?

Let's resolve the thread once the child PR is either merged or closed.

Reply:
Resolving - #258 is merged.